rollout/replicaset.go

```diff
@@ -144,9 +145,8 @@ func (c *rolloutContext) reconcileNewReplicaSet() (bool, error) {
 					newReplicasCount = int32(0)
 				}
 			}
-		} else {
-			abortScaleDownDelaySeconds := time.Duration(defaults.GetAbortScaleDownDelaySecondsOrDefault(c.rollout))
-			err = c.addScaleDownDelay(c.newRS, abortScaleDownDelaySeconds)
+		} else if abortScaleDownDelaySeconds != nil {
```

Comment:
is this condition check required as it is verified in shouldDelayScaleDownOnAbort

Reply:
You're right that it's not required. But I was trying to be defensive here in case there was a bug or change in behavior to shouldDelayScaleDownOnAbort().